[Bug] - error: too many arguments to function 'RzAnalysisFunction'

When building against the latest rizin (commit `171612c5e1e6a8bed2bf2fd3c167c26e4f4db8c0`) rz-retdec will no longer build.

```
/rz-retdec/src/rz-plugin/data.cpp:66:52: error: too many arguments to function 'RzAnalysisFunction* rz_analysis_create_function(RzAnalysis*, const char*, long long unsigned int, RzAnalysisFcnType)'
   66 |                 r2fnc = rz_analysis_create_function(_r2core.analysis, fnc.getName().c_str(),
      |                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   67 |                                 fnc.getStart().getValue(), RZ_ANALYSIS_FCN_TYPE_FCN, nullptr);
      |                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/local/include/librz/rz_parse.h:10,
                 from /usr/local/include/librz/rz_asm.h:12,
                 from /usr/local/include/librz/rz_egg.h:4,
                 from /usr/local/include/librz/rz_core.h:14,
                 from /rz-retdec/include/rz-plugin/data.h:16,
                 from /rz-retdec/src/rz-plugin/data.cpp:11:
/usr/local/include/librz/rz_analysis.h:1305:28: note: declared here
 1305 | RZ_API RzAnalysisFunction *rz_analysis_create_function(RzAnalysis *analysis, const char *name, ut64 addr, RzAnalysisFcnType type);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~
/rz-retdec/src/rz-plugin/data.cpp:75:44: error: too many arguments to function 'bool rz_analysis_fcn_add_bb(RzAnalysis*, RzAnalysisFunction*, long long unsigned int, long long unsigned int, long long unsigned int, long long unsigned int)'
   75 |                 if (!rz_analysis_fcn_add_bb(_r2core.analysis, r2fnc, fnc.getStart().getValue(), fnc.getSize().getValue(), UT64_MAX, UT64_MAX, nullptr))
      |                      ~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/local/include/librz/rz_analysis.h:1518:13: note: declared here
 1518 | RZ_API bool rz_analysis_fcn_add_bb(RzAnalysis *analysis, RzAnalysisFunction *fcn, ut64 addr, ut64 size, ut64 jump, ut64 fail);
      |             ^~~~~~~~~~~~~~~~~~~~~~
make[2]: *** [src/rz-plugin/CMakeFiles/core_retdec.dir/build.make:90: src/rz-plugin/CMakeFiles/core_retdec.dir/data.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:2243: src/rz-plugin/CMakeFiles/core_retdec.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

The following Dockerfile was used.
```Dockerfile
FROM fedora:latest

RUN dnf in -y    python3-devel \
                 python3-pip \
                 cmake \
                 gcc-c++ \
                 gcc \
                 meson \
                 ninja-build \
                 git \
                 zlib-devel \
                 openssl \
                 openssl-devel \
                 python3 \
                 autoconf \
                 automake \
                 libtool \
                 pkg-config \
                 m4 \
                 upx \
                 doxygen \
                 graphviz


RUN git clone https://github.com/rizinorg/rizin.git && \
 cd rizin && \
 meson build && \
 ninja -C build && \
 ninja -C build install 

RUN git clone https://github.com/rizinorg/rz-retdec && \
 cd rz-retdec && \
 mkdir build && cd build && \
 cmake .. -DCMAKE_INSTALL_PREFIX=~/.local && \
 make && \
 make install 
```

--- a/src/rz-plugin/data.cpp
+++ b/src/rz-plugin/data.cpp
@@ -64,7 +64,7 @@ void RizinDatabase::setFunction(const common::Function &fnc) const
 	auto r2fnc = rz_analysis_get_function_at(_r2core.analysis, fnc.getStart().getValue());
 	if (r2fnc == nullptr) {
 		r2fnc = rz_analysis_create_function(_r2core.analysis, fnc.getName().c_str(),
-				fnc.getStart().getValue(), RZ_ANALYSIS_FCN_TYPE_FCN, nullptr);
+				fnc.getStart().getValue(), RZ_ANALYSIS_FCN_TYPE_FCN);
 		if (r2fnc == nullptr) {
 			throw DecompilationError("Unable to create function on address "
 					+ std::to_string(fnc.getStart().getValue()));
@@ -72,7 +72,7 @@ void RizinDatabase::setFunction(const common::Function &fnc) const
 	}
 
 	if (!fnc.isDynamicallyLinked() && fnc.getSize().getValue() > 1)
-		if (!rz_analysis_fcn_add_bb(_r2core.analysis, r2fnc, fnc.getStart().getValue(), fnc.getSize().getValue(), UT64_MAX, UT64_MAX, nullptr))
+		if (!rz_analysis_fcn_add_bb(_r2core.analysis, r2fnc, fnc.getStart().getValue(), fnc.getSize().getValue(), UT64_MAX, UT64_MAX))
 			Log::error() << Log::Warning << "unable to add basic block of " << fnc.getName() << std::endl;
 
 	copyFunctionData(fnc, *r2fnc);

